Makefile

```diff
@@ -72,12 +72,12 @@ tag-release: check-origin-remote
 	git tag -a $(VERSION) -m "Release $(VERSION)"
 	git push origin $(VERSION)
 
-release: tag-release ## package and upload for public release
-	# TODO: Use twine maybe?
-	@echo Honestly, what should this do?
+release: dist ## package and upload for public release
```

Comment:
This is almost what I want, but I think I'd still like to have the `tag-release` target in here. I'd like to be able to run `make release` and have signing, upload to PyPI, and tagging the commit/pushing that to the repo all in one fell swoop.

Reply:
The `dist` target should probably come before `tag-release`.

Reply:
I couldn't verify it, but I think `relase: tag-release dist` would work. Does `make tag-release` hag during `sed` for you?

Reply:
Hmm... it _shouldn't_ hang during `sed`, but it's possible I made an error in that gnarly expression. Let me double check.

Reply:
Nah, pulling the `sed` expression out on its own yields the correct result:
```bash
(layabout) [23:28:15] tucker@khanti:~/Projects/layabout git:(implement-layabout) $ sed -rn "s/__version__ = [\'\"](.*)[\'\"]/\1/p" layabout.py 
1.0.0b
```

Reply:
Heh. I just tried to run this myself though and it's definitely stuck somewhere. I think it's stuck after executing `check-origin-remote`, but before it really gets to `tag-release`...

Reply:
Oh, no, you were right. It's the fact that `$(VERSION_FILE)` is undefined, so `sed` is probably expecting input from stdin. If you swap out `$(VERSION_FILE)` for `$(SRC)` you'll get the right result. This is a classic copy pasta error on my part.